src/packs/walk_directory.rs

```diff
@@ -221,14 +221,9 @@ mod tests {
         let contains_bad_file = included_files.contains(&node_module_file);
         assert!(!contains_bad_file);
 
-        // Although `node_modules/**/*` should probably exclude `node_modules/file.rb`,
-        // it skips the first files in the directory. For now this doesn't affect behavior,
-        // in Gusto's monolith, so keeping as an open bug for now.
-        // To fix this bug, start by changing this test to:
-        // assert!(!contains_bad_file); (instead of assert!(contains_bad_file);)
         let node_module_file = absolute_path.join("node_modules/file.rb");
         let contains_bad_file = included_files.contains(&node_module_file);
```

Comment:
Now that the default configuration issue is fixed, it revealed the this bug was actually fixed in a prior commit!